integration-tests/prepare-operation.spec.ts

```diff
@@ -124,16 +124,21 @@ CONFIGS().forEach(({ lib, setup, protocol, createAddress }) => {
       const receiver = await createAddress();
 
       const pkh = await receiver.signer.publicKeyHash();
-      const preparedTransfer = await Tezos.prepare.transaction({ amount: 1, to: pkh });
+      const estimates = await Tezos.estimate.transfer({ to: pkh, amount: 1 });
+      const preparedTransfer = await Tezos.prepare.transaction({ 
```

Comment:
Because the test does not bother with values of estimates, do we need to make this change? Does all code targeting Taquito need to make the same change?

Reply:
due to the nature of this test, when we prepare & want to send at the same time, we will need to estimate.

Since the test is to test the `preapply` method, we'll need to estimate or the preparation will fail due to incorrect fees.

In a normal circumstance, users would just do something like `Tezos.contract.transfer()`